torchvision/prototype/models/video/resnet.py

```diff
@@ -62,7 +62,7 @@ def _video_resnet(
 class R3D_18Weights(Weights):
     Kinetics400_RefV1 = WeightEntry(
         url="https://download.pytorch.org/models/r3d_18-b3b3357e.pth",
-        transforms=partial(Kinect400Eval, resize_size=(128, 171), crop_size=(112, 112)),
+        transforms=partial(Kinect400Eval, crop_size=(112, 112), resize_size=(128, 171)),
```

Comment:
Changing order to match the other presets.

Reply:
This is just for appearance, right?

Reply:
yes, just for styling. :)